vars/failOnServerError.groovy

```diff
@@ -0,0 +1,8 @@
+/**
+ * grep for errors in server logs throwing an abort exception
```

Comment:
mention that it is in alternative to https://wiki.jenkins.io/display/JENKINS/Text-finder+Plugin not available in pipeline